### PR TITLE
Use `pnpm` and drop `yarn`

### PR DESCRIPTION
**Pull request by TuvalSimha**
General points:

- [ ] Create `pnpm-workspace` according to the root `package.json
- [ ] Not sure what to do with the `Algolia` CI
- [ ] No sure why, maybe because `pnpm`, I had to add packages to `package.json` like here: https://github.com/kamilkisiela/graphql-inspector/pull/2349/files#diff-ba4c6e28ea34b63908b67d16df70b4669b177dc135c0f555b2190e08910b650e
and here
https://github.com/kamilkisiela/graphql-inspector/pull/2349/files#diff-ed7e1388a96685c21728af0c7c6b15d4d9bee7001a902455adff6c25a470d4e7
- [ ] Should I run `changesets` for this PR?

### CODE REVIEW

**Review by dotansimha — COMMENTED** [review]
@TuvalSimha for dependencies inside that monorepo, you should use `workspace:*` version instead of the numbered version, otherwise you'll have issues in the future with releases

File: Dockerfile [hunk 1]
```diff
@@ -6,8 +6,8 @@ WORKDIR /app
 
 COPY . .
 
-RUN yarn install
-RUN yarn build
+RUN pnpm
```
Comment (kamilkisiela) [inline Dockerfile:9]:
```suggestion
RUN pnpm install
```

**Review by kamilkisiela — CHANGES_REQUESTED** [review]
One small thing to change but once it's done, let's merge it

File: Dockerfile (hunk 1 above)
Comment (dotansimha) [reply Dockerfile:9]: also, I think this one is going to fail? 
because default node image does not have `pnpm` by default?

**Comment by TuvalSimha** [comment]
> @TuvalSimha for dependencies inside that monorepo, you should use `workspace:*` version instead of the numbered version, otherwise you'll have issues in the future with releases

Like that, no?

https://github.com/kamilkisiela/graphql-inspector/pull/2349/files#diff-18ae0a0fab29a7db7aded913fd05f30a2c8f6c104fadae86c9d217091709794c

**Comment by dotansimha** [comment]
>  Like that, no?
> 
> [#2349 (files)](https://github.com/kamilkisiela/graphql-inspector/pull/2349/files#diff-18ae0a0fab29a7db7aded913fd05f30a2c8f6c104fadae86c9d217091709794c)

No, in `package.json` of the actual packages. For example https://github.com/kamilkisiela/graphql-inspector/pull/2349/files#diff-6e2e2a1851648938b325ba84de634407a4e69a644ea61102df15ca4a8a7a9758R50-R66 , it needs to use `workspace:*` instead of the versioned specified there.

**Comment by TuvalSimha** [comment]
> > Like that, no?
> > [#2349 (files)](https://github.com/kamilkisiela/graphql-inspector/pull/2349/files#diff-18ae0a0fab29a7db7aded913fd05f30a2c8f6c104fadae86c9d217091709794c)
> 
> No, in `package.json` of the actual packages. For example https://github.com/kamilkisiela/graphql-inspector/pull/2349/files#diff-6e2e2a1851648938b325ba84de634407a4e69a644ea61102df15ca4a8a7a9758R50-R66 , it needs to use `workspace:*` instead of the versioned specified there.

@dotansimha 
got it, i did new changes

File: .changeset/@graphql-inspector_cli-2349-dependencies.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
+"@graphql-inspector/cli": patch
+---
+dependencies updates:
```
Comment (dotansimha) [inline .changeset/@graphql-inspector_cli-2349-dependencies.md:4]: These should be deleted, it's a bug in the tool that creates the automatic changesets :/ I'll fix and then you can delete those

Comment (dotansimha) [reply .changeset/@graphql-inspector_cli-2349-dependencies.md:4]: ok fixed, you can remove these and it should not create it again

**Pull request merged**